plugins/WebsiteMeasurable/MeasurableSettings.php

```diff
@@ -84,6 +85,10 @@ public function __construct(SitesManager\API $api, Plugin\Manager $pluginManager
 
     protected function init()
     {
+        if ($this->idMeasurableType !== Type::ID && $this->idMeasurableType !== MobileAppType::ID) {
+            return;
+        }
+
```

Comment:
I will use inheritance for now, but usually inheritance is a bit of a problem and rather not such a good solution but for now it will do